gulpfile.js

```diff
@@ -73,6 +74,7 @@ function build()
         .pipe(addsrc.append(['lib/waves/**/test-object.js']))
         .pipe(addsrc.prepend(['license.txt', 'lib/mojito.js', 'lib/shared-code.js']))
         .pipe(concat(containerName + '.pretty.js'))
+        //.pipe(babel({presets: [['@babel/preset-env', { "modules": false, exclude:['@babel/plugin-transform-typeof-symbol'] }]]}))
```

Comment:
Guessing that if we want transpiling before uglifying, we just uncomment this line?

Reply:
@dapperdrop 
Ah, my bad, I forgot to uncomment that line after the last test. Now I have submit a new commit.